### DISCUSSION TITLE
(filezilla)

### DISCUSSION THREAD

**Discussion by uwemolnar** [chocolatey-community/chocolatey-packages#1880] [Ideas]
As requested by the maintainer of filezilla: the installer-links for filezilla seem to be outdated. Can you please check and fix them:
https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/filezilla/legal/VERIFICATION.txt

Would be great - thanks!

**Comment by AdmiringWorm** [comment]
hmm, this is a bit problematic to get updated.

Sure, we could update the URL used, but to push it to the Chocolatey Community Repository, we must create a fix version (IE, a version of `3.59.0.20220510`).
Considering that the links seem to be timed, this would mean we'd need to create a fix every time the URL changes (which is what, daily, hourly, ....).

The links specified are the links that were used when the package was created, but since they are timed links, it seems there are some enhancements to the VERIFICATION.txt file we need to do instead.

See the issue that was created from this discussion: #1881

**Reply by uwemolnar** [reply]
Okay, I see the problem! 

Looks like there is a (Unix)-timestamp included in the URL. Furthermore different servers are used and there are also (cryptographic?) strings added. That makes it for sure quiet impossible. 

Nevertheless thanks for your answer!
